### PR TITLE
feat: Enable TP item deployment

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR enables the deployment of TP items.
The deployment of TP items can be tested in `peer-uw-1.decentraland.zone`.

### CODE REVIEW

File: src/modules/collection/sagas.ts [hunk 1]
```diff
@@ -628,7 +627,7 @@ export function* collectionSaga(builder: BuilderAPI, catalyst: CatalystClient) {
       }
 
       // Open the ApprovalFlowModal with the items to be approved
-      // 4. Make the transaction to the contract (update of the merkle tree root with the signature and its parameters)
+      // 4. Make the transaction to the contract (update of the merkle tree root with the signature and its parameters)\
```
Comment (nicosantangelo) [inline src/modules/collection/sagas.ts:630]: 👀

File: src/modules/item/export.ts [hunk 2]
```diff
@@ -153,14 +154,14 @@ function buildItemEntityMetadata(collection: Collection, item: Item): StandardCa
     name: item.name,
     description: item.description,
     collectionAddress: collection.contractAddress!,
-    rarity: item.rarity!,
-    i18n: [{ code: 'en', text: item.name }],
+    rarity: (item.rarity! as unknown) as Rarity,
+    i18n: [{ code: 'en', text: item.name }] as I18N[],
```
Comment (nicosantangelo) [inline src/modules/item/export.ts:158]: maybe we can save the casting here and use `Locale` everywhere?

```
i18n: [{ code: Locale.EN, text: dbItemMock.name }],
```

File: src/modules/item/export.ts [hunk 3]
```diff
@@ -153,14 +154,14 @@ function buildItemEntityMetadata(collection: Collection, item: Item): StandardCa
     name: item.name,
     description: item.description,
     collectionAddress: collection.contractAddress!,
-    rarity: item.rarity!,
-    i18n: [{ code: 'en', text: item.name }],
+    rarity: (item.rarity! as unknown) as Rarity,
```
Comment (nicosantangelo) [inline src/modules/item/export.ts:157]: Same question as [here](https://github.com/decentraland/builder-server/pull/470#discussion_r839875557) in the server. If we won't change anything there, dismiss!

**Review by nicosantangelo — COMMENTED** [review]
🔥

File: src/modules/item/export.ts (hunk 2 above)
Comment (LautaroPetaccio) [reply src/modules/item/export.ts]: Done!

**Review by nicosantangelo — APPROVED** [review]
+1

**Pull request merged**